tests/sentry/api/test_id_or_slug_path_parameters.py

```diff
@@ -228,6 +264,108 @@ def region_sentry_app_test(self, endpoint_class, slug_params, non_slug_params, *
+    )
+    @override_options({"api.id-or-slug-enabled": True})
+    def organization_search_details_test(self, endpoint_class, slug_params, *args):
+        slug_kwargs = {param: self.slug_mappings[param].slug for param in slug_params}
+        id_kwargs = {param: self.slug_mappings[param].id for param in slug_params}
+
+        request = Request(request=self.make_request())
+        search = self.create_saved_search(name="test", organization=self.organization)
+
+        non_slug_mappings: dict[str, Any] = {
+            "search_id": search.id,
+        }
+
+        # mapping kwargs to the actual objects
+        reverse_non_slug_mappings: dict[str, Any] = {
+            "search": search,
+        }
+
+        _, converted_slugs = endpoint_class().convert_args(
+            request=request, **slug_kwargs, **non_slug_mappings
+        )
+        _, converted_ids = endpoint_class().convert_args(
+            request=request, **id_kwargs, **non_slug_mappings
+        )
+
+        self.assert_conversion(
+            endpoint_class, converted_slugs, converted_ids, reverse_non_slug_mappings
+        )
+
+    @patch("sentry.api.bases.organizationmember.OrganizationMemberEndpoint._get_member")
+    @patch(
+        "sentry.api.endpoints.organization_member.requests.invite.details.OrganizationInviteRequestDetailsEndpoint._get_member"
+    )
+    @patch("sentry.api.bases.organization.OrganizationEndpoint.check_object_permissions")
+    @patch("sentry.types.region.subdomain_is_region")
+    @override_options({"api.id-or-slug-enabled": True})
+    def organization_member_test(self, endpoint_class, slug_params, *args):
+        slug_kwargs = {param: self.slug_mappings[param].slug for param in slug_params}
+        id_kwargs = {param: self.slug_mappings[param].id for param in slug_params}
+
+        request = Request(request=self.make_request(user=self.user))
+
+        non_slug_mappings: dict[str, Any] = {
+            "member_id": self.user.id,
+        }
+
+        _, converted_slugs = endpoint_class().convert_args(
+            request=request, **slug_kwargs, **non_slug_mappings
+        )
+        _, converted_ids = endpoint_class().convert_args(
+            request=request, **id_kwargs, **non_slug_mappings
+        )
+
+        # Remove member from the converted slugs and ids, we don't need to compare it b/c its mocked
+        converted_slugs.pop("member")
+        converted_ids.pop("member")
+
+        self.assert_conversion(endpoint_class, converted_slugs, converted_ids)
+
+    @patch("sentry.api.bases.group.GroupEndpoint.check_object_permissions")
+    @override_options({"api.id-or-slug-enabled": True})
+    def group_test(self, endpoint_class, slug_params, *args):
+        slug_kwargs = {param: self.slug_mappings[param].slug for param in slug_params}
+        id_kwargs = {param: self.slug_mappings[param].id for param in slug_params}
+
+        request = Request(request=self.make_request())
```

Comment:
Do we need to cast this for typing?

Reply:
make_request makes an `HttpRequest` obj but I need a `Request` obj so I instantiate a Request object and set its request field to the `HttpRequest`.